changelogs/current.yaml

```diff
@@ -210,5 +210,11 @@ new_features:
     and
     :ref:`response_attributes <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExternalProcessor.response_attributes>`
     config APIs to enable sending and receiving attributes from/to the external processing server.
+- area: udp_proxy
+  change: |
+    Delete session filters before write session access log to allow destructors of session filters to run and write to stream
+    info before writing session access log.
+    This behavioral change can be temporarily reverted by setting runtime guard
+    ``envoy.reloadable_features.delete_session_filters_before_write_session_access_log`` to false.
```

Comment:
I wasn't sure we need runtime flag here, since I don't know why people want to revert back. But I leave this to senior mantainer to make decision.